### ISSUE TITLE
Filter GNOME apps

### ISSUE THREAD

**Issue by cassidyjames** [elementary/appcenter-web#75]
Web and Evince show up since they are available in the remote, but they are not really the same as other curated AppCenter apps.

**Comment by dar5hak** [comment]
I could give this a shot in a day or two.

BTW what is the rationale for showing first-party elementary apps (like Calculator) here? I get that they show up in the GTK AppCenter, but so do Web and Evince.

**Comment by cassidyjames** [comment]
@dar5hak they're first-party apps shipped as Flatpak in AppCenter, so it makes sense to show them. In the future we could even ship new but not-preinstalled apps in AppCenter.

I would be open to showing the GNOME apps as well if we rebranded them as we do in the OS itself, as these apps are not really the same as the upstream versions. To do so, we'd need to ship updated AppStream for them, which would fix them in the GTK client as well.

So I'm okay with either approach, but filtering them here seems like a quicker first step until we can sort it out on the AppStream side.

**Comment by dar5hak** [comment]
Makes sense. I see the possibility where some first-party app isn’t a candidate for being pre-installed, say, something targeted at developers.

**Comment by dar5hak** [comment]
I haven't been able to work on this, but for anyone who wants to try, these sources will be helpful:

- https://stackoverflow.com/a/64120329/14499752
- https://www.webisland.agency/blog/jekyll-remote-assets-linking-filter/

**Comment by dar5hak** [comment]
I haven't been able to work on this, but for anyone who wants to try, these sources will be helpful:

- https://stackoverflow.com/a/64120329/14499752 because Liquid doesn't come with a built-in `starts_with` filter
- https://www.webisland.agency/blog/jekyll-remote-assets-linking-filter/ as a guide to creating a custom filter

**Comment by dar5hak** [comment]
No luck with the custom filter.

I tried with `contains` but apparently Liquid doesn't have a straightforward `not` operator, so you can do this:

```liquid
flatpak_apps | where_exp:"app","app.id contains 'org.gnome'"
```

but not this:

```liquid
flatpak_apps | where_exp:"app","not app.id contains 'org.gnome'"
```
